src/main/java/com/stories/domain/StoryDomain.java

```diff
@@ -0,0 +1,26 @@
+package com.stories.domain;
+
+import java.util.ArrayList;
+import java.util.Date;
+import lombok.Data;
+
+@Data
+public class StoryDomain {
+
```

Comment:
Add no-args constructor to initialize history to a new `ArrayList `of tipe `String`.